server/build.gradle

```diff
@@ -61,6 +61,7 @@ dependencies {
     compile "javax.xml.bind:jaxb-api:${versions.jaxb_api}"
 
     testCompile project(path: ':libs:dex', configuration: 'testOutput')
+    testCompile project(':libs:shared')
```

Comment:
is it needed? `testCompile` extends compile, there is already `compile project(':libs:shared')`

Reply:
You are right, this was local setup problem. Thanks!

Reply:
It is removed from the commit already.